src/test/java/uk/gov/companieshouse/extensions/api/requests/RequestControllerUnitTest.java

```diff
@@ -1,22 +1,15 @@
 package uk.gov.companieshouse.extensions.api.requests;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.anyString;
 import static org.mockito.ArgumentMatchers.eq;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
-import static uk.gov.companieshouse.extensions.api.Utils.Utils.BASE_URL;
-import static uk.gov.companieshouse.extensions.api.Utils.Utils.COMPANY_NUMBER;
-import static uk.gov.companieshouse.extensions.api.Utils.Utils.EMAIL;
-import static uk.gov.companieshouse.extensions.api.Utils.Utils.FORENAME;
-import static uk.gov.companieshouse.extensions.api.Utils.Utils.SURNAME;
-import static uk.gov.companieshouse.extensions.api.Utils.Utils.USER_ID;
-import static uk.gov.companieshouse.extensions.api.Utils.Utils.dummyRequestDTO;
-import static uk.gov.companieshouse.extensions.api.Utils.Utils.dummyRequestEntity;
+import static uk.gov.companieshouse.extensions.api.Utils.Utils.*;
```

Comment:
Need specific imports here

Reply:
Thought I'd got all these the ide keeps on doing this to try and be helpful

Reply:
Are you using intellij? You can tell it to stop doing this, 
Preferences -> Editor -> Code Style -> Java
Select "Use single class import" and set "class count to use import with '*' and Name count to use static import with '*' to 99. And it should stop adding wildcards.